src/Analyser/MutatingScope.php

```diff
@@ -674,6 +674,10 @@ private function getNodeKey(Expr $node): string
 			$key .= '/*' . $node->getAttribute('startFilePos') . '*/';
 		}
 
+		if ($node->hasAttribute('keepVoid')) {
```

Comment:
would be great if we could put this string into a class-constant somewhere.

that way it would ease navigation with the IDE and make dependent parts obvious

Reply:
yeah, I wanted to check first is all is fine and then ask about how to properly refactor this thing ☺️ I hate that it is sprinkled x times around the code base now..

Reply:
Could be a new method on Scope, just don't start it with `getType`, I'd hate to confuse autocompletion with this niche use case

Reply:
still need to do this refactor, missed the comment. currently working on it